app/database/models/mentorship_relation.py

```diff
@@ -5,6 +5,24 @@
 
 
 class MentorshipRelationModel(db.Model):
+    """A relatioal model the represents a mentorship.
```

Comment:
`"""Data Model representation of a mentorship relation.` 
This should be better. Refer to: https://github.com/systers/mentorship-backend/pull/184/files/5e89077a0ed0dab3ba31bacf2b8ca3d28edc3def#r252468522